Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -0,0 +1,550 @@
+import CTensorFlow
+
+/// The `TF_Tensor *` type.
+typealias CTFTensor = OpaquePointer
+
+public class LazyTensor : _AnyTensorHandle {
```

Comment:
Does `LazyTensor` need to be public too?

Reply:
No, but I need to mark it usableFromInline for some changes that I am working on.